realm/src/main/java/io/realm/internal/LinkView.java

```diff
@@ -30,58 +33,52 @@ public LinkView(Context context, Table parent, long columnIndexInParent, long na
         this.nativeLinkViewPtr = nativeLinkViewPtr;
     }
 
-    protected static native void nativeClose(long nativeLinkViewPtr);
```

Comment:
I don't see the need to reorder the private methods. It's easier to have it just where it's used when you change it. And it's consistent with other files. (alternatively other files should also be reordered - which I don't think is warranted)